contracts/libraries/PriorityQueue.sol

```diff
@@ -1,44 +1,58 @@
 pragma solidity ^0.4.24;
 
+// external modules
 import "openzeppelin-solidity/contracts/math/SafeMath.sol";
-import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
 
-/**
- * @title PriorityQueue
- * @dev A priority queue implementation
- */
-
-contract PriorityQueue is Ownable {
+library PriorityQueue {
```

Comment:
Can you quickly add a comment saying that the list has an unused element at index 0.

Reply:
pushed a comment.